src/secrules_parsing/model/secrules.tx

```diff
@@ -34,7 +34,7 @@ SecRuleRemoveByTag:
 
 // ActionList can be empty if is the last rule in chain    
 SecRule:
-    'SecRule' variables+=Variable['|'] '"' negated='!'? '@'? operator=Operator '"' '"'? actions+=Action[',']? '"'?;
+    'SecRule' variables+=Variable['|'] '"' negated='!'? '@'?- operator=Operator '"' '"'? actions+=Action[',']? '"'?;
```

Comment:
From what I understand, `Operator` would include the `@` if it was there and otherwise not. So now it will never include the `@`, is that correct? If so, I would expect a change in the parser too, since there probably is an explicit check for the existence of the `@` in the operator name.

Reply:
No, `Operator` didn't include it. The `@` here wasn't saved anywhere, so it wasn't being used. The `-` now specifically discards it.